tests/h/tweens_test.py

```diff
@@ -15,7 +16,9 @@ def test_it_loads_redirects(self, patch):
 
         tweens.redirect_tween_factory(handler=None, registry=None)
 
-        open_.assert_called_once_with("h/redirects", encoding="utf-8")
+        open_.assert_called_once_with(
+            Any.string.matching("^.*h/redirects$"), encoding="utf-8"
```

Comment:
This is the other side of the allowing functests to be run in any directory. This for me is `/home/me/blah/.../h/redirects`